components/brave_extension/extension/brave_extension/background/api/shieldsAPI.ts

```diff
@@ -112,7 +113,7 @@ export const setAllowTrackers = (origin: string, setting: string) =>
  * @return a promise which resolves when the setting is set
  */
 export const setAllowHTTPUpgradableResources = (origin: string, setting: BlockOptions) =>
-  chrome.braveShields.setHTTPSEverywhereControlTypeAsync(setting, origin)
+  chrome.braveShields.setHTTPSEverywhereEnabledAsync(setting == 'allow' ? false : true, origin)
```

Comment:
the `allow`/`block` toggles were always confusing but with this PR having main shields and httpsE controls using booleans make it much more confusing.

I don't think this is a blocker for this PR since it's a crucial fix, but can we have a follow-up task to have all controls set as a boolean?

Reply:
not all controls make sense as booleans. I discussed with @bbondy and @SergeyZhukovsky and we thought using Enabled for these made it easier to understand vs trying to figure out what the correct allow/block setting was for them. With cookies, etc.. it's much more intuitive